### PR TITLE
React Admin - Search/Filters/Ordering for Post/Competition/Semester/Series/Problems

### LINKED ISSUE #264: RA - All lists - Filtrovanie, potencialne Search?
Ked budeme mat viac dat, bude sa v nich treba lepsie vyznat, vediet vyfiltrovat based on competition atd.

### PR DESCRIPTION
**Pull request by rtrembecky**
~potrebuje https://github.com/ZdruzenieSTROM/webstrom-backend/pull/460~

~berme to ako WIP, treba overit este dalsie BE modely a mergnut BE, kym moze toto ist tiez~

resolves #264

### CODE REVIEW

**Comment by kovacspe** [comment]
Odstránil som label waiting-for-backend lebo už je mergnutá podpora na backende

File: src/components/Admin/resources/competition/semester/SemesterList.tsx [hunk 1]
```diff
@@ -11,35 +11,50 @@ import {
...
         render={(record) => record && <span>{record['late_tags'].length}</span>}
+        sortable={false}
       />
     </Datagrid>
   </List>
 )
+
+const SemesterListFilters: FC = () => (
+  <FilterSidebar>
+    <CompetitionFilterSection />
```
Comment (Matushl) [inline src/components/Admin/resources/competition/semester/SemesterList.tsx:54]: tu by sme to mali iba podla seminara filtrovat, nie polda competition

alebo to aspon prefiltrovat len na tie hodnoty co davaju zmysel nakolko competitions je plna prdel (to iste pre `competition/series` a `competition/problems`

**Review by Matushl — APPROVED** [review]
V podstate je tu super, ale bude to chciet este dolatit a mozno upravit pre niektore stranky

File: src/components/Admin/resources/competition/semester/SemesterList.tsx (hunk 1 above)
Comment (rtrembecky) [reply src/components/Admin/resources/competition/semester/SemesterList.tsx:54]: ez 
image

**Pull request merged**